diem-move/diem-framework/DPN/sources/DiemAccount.move

```diff
@@ -546,6 +548,20 @@ module DiemFramework::DiemAccount {
         }
     }
 
+    public fun vm_create_account_migration(
+        vm: &signer,
+        new_account: address,
+        new_account_authkey_prefix: vector<u8>,
+        // value: u64,
+    ) acquires AccountOperationsCapability  {
+        CoreAddresses::assert_diem_root(vm);
+        // TODO: fix the timestamp issue
```

Comment:
@0o-de-lally can you comment on the timestamp issue? There is no common timestamp in the form of height since genesis?

Reply:
There's a bug in this version of Diem. @Corynthian had pointed it out in another issue. Would like to pull that change into V6. Doesn't have to be on this branch.